content/workers/platform/bindings/about-service-bindings.md

```diff
@@ -0,0 +1,94 @@
+
+![To configure a Service binding, head to your Worker's Settings tab](../media/service-bindings-config.png)
+
+Workers bound to your Worker will be listed in the [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Triggers** tab, under **Bound Services**. Your team can easily view cross-service dependencies in this manner.
+
+![Your team can view cross-service dependencies in the Triggers tab](../media/service-bindings-triggers.png)
+
+## Composing an example Worker
+
+In the following example, we'll create a `gateway` Worker that invokes an `auth` Worker to handle authorization checks.
+
+### Gateway Worker and Service bindings usage
+
+First, we'll create both a `gateway` and `auth` Worker. Once they're set up, navigate to the `gateway` Worker. In order to bind and call the [authentication](#authentication-workers-service) Worker, the `gateway` Worker needs to set up a Service binding. You can manage Service bindings in [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Settings**> **Variables** > **Service bindings** > **Edit variables**.
+
+![Selecting Edit variables to create new bindings and edit existing bindings that enable Worker-to-Worker communication](../media/service-bindings.png)
+
+Once added, the `gateway` Worker can access the Workers Service binding directly from the code, as in the example below. It utilizes the `fetch` API.
+
+```js
+export default {
+  async fetch(request, env) {
+    // Fetch AUTH service and pass request
+    const authResponse = await env.auth.fetch(request.clone());
+
+    // Return response from the AUTH service if the response status is not 200
+    // It would return 403 'x-custom-token does not match, request not allowed' response in such case
+    if (authResponse.status !== 200) {
+      return authResponse;
+    }
+
+    // Request allowed
+    // You can write application logic here
+    // In this case we delegate the logic to an `application` Worker
+    return await env.application.fetch(request)
+  },
+};
+```
+
+{{<Aside type="note">}}
+
+Note that [Requests](../../runtime-apis/request) can only be read once. If you need to use a Request object multiple times, clone your incoming Request objects.
+
+{{</Aside>}}
+
+In this setup, only the `gateway` Worker is exposed to the Internet and privately communicating with the `auth` and `application` Workers using Service bindings. Read on to build the `auth` Worker.
+
+### Authentication Workers Service
+
+The following authentication Workers Service code responds with `200` in the case that `x-custom-token` in the incoming request matches a `SECRET_TOKEN` Secret binding. Note that we implement `fetch` here, since a Service binding will invoke FetchEvent on the target Worker.
```

Comment:
```suggestion
The following authentication Workers Service code responds with a status code `200` in the case that `x-custom-token` in the incoming request matches a `SECRET_TOKEN` secret binding. Note that you implement `fetch` here, since a Service binding will invoke FetchEvent on the target Worker.
```